### ISSUE TITLE
/home/circleci/.local/bin is in path but doesn’t exist by default

### ISSUE THREAD

**Issue by aral** [CircleCI-Public/cimg-node#286]
While apps/scripts should ideally check that `~/.local/bin` exists before trying to do something with it, it might be an idea to have the directory exist by default in the image given that it’s already in `$PATH`.

(Update: Note, it’s the `~/.local` directory that doesn’t exist.)

(Thanks, by the way, this image is _hugely_ useful.) :)

**Comment by felicianotech** [comment]
Hi @aral,

Thank you for bringing this up and I'm SO happy the image has been useful for you.

When originally setting things up, we learned that the directories in `PATH` don't need to exist according to standard. However, I agree that there's no reason to not create them ahead of time. Where this change needs to occur (and was merged) is in https://github.com/CircleCI-Public/cimg-base/pull/205

The change will land in the October snapshot of the base image. Once that happens, it will make its way into this Node.js image later that month.
